3561/3/.gitignore

```diff
@@ -0,0 +1,3 @@
+.env
+.idea
```

Comment:
JFYI: Better way is using [global gitignore](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html) for IDE files